training-node/app/models/user.js

```diff
@@ -0,0 +1,11 @@
+module.exports = (sequelize, DataTypes) => {
```

Comment:
there's no need of the `const user` and the return, you can simply return the `sequelize.define` if you remove the brackets, like a function